HR

```diff
@@ -0,0 +1 @@
+
```

Comment:
@kristinaleko AFAIC you could just delete this PR altogether.

Reply:
Hi Meinte, I wanted to delete all the previous uploads and to upload new files. Currently whn I try to drag and drop the new HR folder with test rules I get the following message: **"Something went really wrong, and we can't process that file."**

I have no idea what could be wrong , we have changed the language also but still i get the same message. 

update: I have tried to upload now again for the secont time and now I just see progress bar stuck on: **"Uploading 1 of 79 files"**

can you help?

Reply:
@kristinaleko Hi Kristina, I'm a bit confused about the use of the word “upload”. Do you mean the commits that are part of this Pull Request, or the uploads to the EU DCC Gateway?

Reply:
@dslmeinte upload of rule test examples here to this github repository, to my fork so that the examples can be merged into main:

Reply:
@kristinaleko Could you try to merge *from* the original repo first? I see on your forked repo you're 59 commits behind, which might explain things.

Reply:
@dslmeinte did it, but still the same error appears: "Something went really wrong, and we can't process that file"

maybe this has to do with it: "This branch is 4 commits ahead of eu-digital-green-certificates/dgc-business-rules-testdata:main"

can you see if everything is pushed, pulled what it should be? 
i'm not sure anymore :)

Reply:
@kristinaleko Commits ahead, but none behind is what you want for a PR, so that's not it. I'll have a better look at your repo as soon as I can - daddy day care day, today.

Reply:
@dslmeinte hi meinte, hope that the daddy day was nice :)
will you be able today to check the upload error? I can send you via email the rar file with test rules that we have prepared if necessery?

Reply:
@kristinaleko Daddy day care days are nice, but generally also more tiresome than work days ;)

I'm a bit confused by the error message because it doesn't seem to fit to pushing your branch to GitHub. You say you're now 4 commits ahead while https://github.com/kristinaleko/dgc-business-rules-testdata says 3 commits ahead, so I'm guessing you have an unpushed commit. Does the error occur when pushing that branch to GitHub? If so, have you tried force-pushing?